python/share/modules/Python/__init__.py

```diff
@@ -7,9 +7,17 @@
 from math import *
 import os
 
+try:
```

Comment:
Actualy I had this before but I dropped it because it boost::python segaulted on reloads. I have to check if pybind11 sgvs too.

Reply:
Yes it does. Well I dont know what to do here. Either the app segfaults when the python interpreter is reloaded or I have to review all import. Ill have to sleep a night over it. Maybe integrating the python extension in the core, which forces the python interpreter to never be unloaded, is an idea. It would solve other problems too but its really a ton of work.

Reply:
Maybe related to https://github.com/albertlauncher/albert/issues/540 ?